modules/virt-connecting-vm-secondarynw-using-fqdn.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc
+
+:_content-type: PROCEDURE
+[id="virt-connecting-vm-secondarynw-fqdn_{context}"]
+= Connecting to a virtual machine on a secondary network by using the cluster FQDN
+
+You can access a virtual machine (VM) that is attached to a secondary network interface from outside the cluster by using the fully qualified domain name (FQDN) of the cluster.
+
+.Prerequisites
+* The QEMU guest agent must be running on the virtual machine.
+* The IP address of the VM that you want to connect to, by using a DNS client, must be public.
+* You have configured the DNS server for secondary networks.
+* You have retrieved the fully qualified domain name (FQDN) of the cluster.
+
+.Procedure
+. Retrieve the virtual machine instance (VMI) configuration by using the following command:
++
+[source,terminal]
+----
+$ oc get vmi -n secondary-test vmi-sec -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  name: vmi-sec
+  namespace: secondary-test
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - bridge: {}
+        name: nic1
+    resources:
+      requests:
+        memory: 128Mi
+  networks:
+  - multus:
+      networkName: ptp-conf
+    name: nic1
+  volumes:
+  - containerDisk:
+      image: quay.io/kubevirt/cirros-container-disk-demo:devel
+      imagePullPolicy: Always
+    name: containerdisk
+  - cloudInitNoCloud:
+      userData: |
+        #!/bin/sh
+        echo 'printed from cloud-init userdata'
+    name: cloudinitdisk
```

Comment:
In other examples we use '...' instead of unrelated details. We could simplify this example to something similar to https://docs.openshift.com/container-platform/4.12/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html#virt-attaching-vm-additional-network-cli_virt-attaching-vm-multiple-networks